conf/test_alignment_to_fastq.config

```diff
@@ -0,0 +1,15 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/sarek -profile test,<extra_test_profile>,<docker/singularity> --outdir <OUTDIR>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+params {
+    input                  = "${projectDir}/tests/csv/3.0/bam_for_remapping.csv"
+    tools                  = null
```

Comment:
```suggestion
    input = "${projectDir}/tests/csv/3.0/bam_for_remapping.csv"
    tools = null
```

Reply:
Yep, but to keep on top of this you would need to adjust all files if the spacing changes in any one file. This will create an unnecessary diff which is why it may be better to just maintain this at the file level?

Reply:
You're totally right, just trying to justify ;-) the choice.
I'll adjust it while working on fixing the last failing test.

We did encounter some unexpected interaction with this new profile inclusion that's under investigation